examples/kubeflow/README.md

```diff
@@ -0,0 +1,255 @@
+
+For more about service definitinos, please see [Service and APIs](https://docs.bentoml.org/en/latest/concepts/service.html).
+
+## Build Service
+
+Building the service and models into a bento allows it to be distributed among collaborators, containerized into a OCI image, and deployed in the Kubernetes cluster. To build a service into a bento, we first need to define the `bentofile.yaml` file. See [Building Bentos](https://docs.bentoml.org/en/latest/concepts/bento.html) for more options.
+
+```yaml
+service: "service:svc"
+include:
+- "service.py"
+- "sample.py"
+python:
+  requirements_txt: ./requirements.txt
+```
+
+Running the following command will build the service into a bento and store it to the local bento store.
+
+```bash
+bentoml build
+```
+
+## Serve Bento
+
+Serving the bento will bring up a service endpoint in HTTP or gRPC for the service API we defined. Use `--help` to see more serving options.
+
+```bash
+bentoml serve-http --production
+```
+
+## Containerize and Push Image
+
+Containerize the image through `docker build` and `push` the image to a remote repository of your choice.
+
+```bash
+bentoml containerize fraud_detection:zsk3powbr2adgcvj -t your-username/fraud_detection:zsk3powbr2adgcvj
+```
+
+```bash
+docker push your-username/fraud_detection:zsk3powbr2adgcvj
+```
+
+## Deploy to Kubernetes Cluster
+
+BentoML offers three custom resource definitions (CRDs) in the Kubernetes cluster.
+
+- [BentoRequest](https://docs.bentoml.org/projects/yatai/en/latest/concepts/bentorequest_crd.html)
+- [Bento](https://docs.bentoml.org/projects/yatai/en/latest/concepts/bento_crd.html)
+- [BentoDeployment](https://docs.bentoml.org/projects/yatai/en/latest/concepts/bentodeployment_crd.html)
+
+Since we have already built and pushed image in the earlier step, we will only need to create the `Bento` and `BentoDeployment` custom resources. The `Bento` CRD describes the metadata for the Bento such as the address of the image and the runners. The `BentoDeployment` CRD describes the metadata of the deployment such as resources and autoscaling behaviors. `BentoDeployment` requires `Bento` to be ready and will reconcile a `Bento` CR of the same name before a deployment is created. See `deployment.yaml` file included for an example.
+
+The `BentoRequest` CRD describes the meta data needed for building the image such as the download URL of the Bento. Creating the `BentoRequest` CR is not needed in this example.
```

Comment:
Should we mention the image-builder setup and link to related docs?

Reply:
For this example, I'm thinking to keep it simple and not mention the more complex `image-builder` custom installation. After we add the image builder custom installation documentation to the Kubeflow documentation, we can link it there and mention that Kubeflow can build the image for you.

Reply:
Added both deployment workflows with `BentoRequest` and `Bento`.